10.9-libcxx/stable/main/finkinfo/sci/r-base40.info

```diff
@@ -0,0 +1,375 @@
+  install_name_tool -id %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/modules/R_de.so R_de.so
+  install_name_tool -id %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/modules/R_X11.so R_X11.so
+  popd
+  
+  mkdir -p bin
+  pushd bin
+  ln -s %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/bin/R R
+  ln -s %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/bin/R R-%type_raw[rversion]
+  popd
+  
+  mkdir -p share/man/man1
+  pushd share/man/man1
+  ln -s %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/man1/R.1 R.1
+  popd
+  
+  # Keep the library folder safe. Users can install their libraries in $HOME/Library/R.
+  chmod 755 Library/Frameworks/R.framework/Resources/library
+  
+  # Satisfy validation
+  rm -rf Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/library/Matrix/include
+
+  # AKH: fix install_names
+  cd %i/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/library
+  install_name_tool -id \
+  		%p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/library/grDevices/libs/cairo.so \
+ 		grDevices/libs/cairo.so
+
+  # Fix the framework path
+  perl -pi -e "s|LIBR =|LIBR = -L%p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/lib -lR #|" %i/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/etc/Makeconf
+
+  # dmacks: Don't force all modules to inherit flags it at their build
+  # time (internal-use only, not needed to be visible by others)
+  perl -pi -e 's/^(LIBS\s*=)/\1 #/g' %i/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/etc/Makeconf
+  
+  # Fix the permissions
+  chmod -R g-w %i
+<<
+RunTimeVars: <<
+  R_HOME: %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources
+<<
+
+SplitOff: <<
+  Package: %N-shlibs
+  Replaces: <<
+		%N
+  <<
+  Description: R Libraries
+  Depends: <<
+	bzip2-shlibs,
+	cairo-shlibs (>= 1.12.14-1),
+	gcc9-shlibs,
+	glib2-shlibs (>= 2.22.0-1),
+	libcurl4-shlibs,
+	libgettext8-shlibs,
+	libiconv,
+	libicu55-shlibs,
+	libjpeg9-shlibs,
+	liblzma5-shlibs,
+	libpng16-shlibs,
+	libtiff5-shlibs,
+	libxt-shlibs,
+	pango1-xft2-ft219-shlibs (>= 1.24.5-4),
+	libpcre1-shlibs, libpcre2-shlibs,
+	readline8-shlibs,
+	tcltk
+  <<
+  Files: <<
+  Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/etc/Renviron
+  Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/etc/ldpaths
+  Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/lib
+  Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/library
+  Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/modules
+  <<
+  Shlibs: <<
+  %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/lib/libR.dylib 4.0.5 %n (>= 4.0.5-1)
```

Comment:
```suggestion
  %p/Library/Frameworks/R.framework/Versions/%type_raw[rversion]/Resources/lib/libR.dylib 4.0.0 %n (>= 4.0.5-1)
```
see comment on the validation error. The actual starting version for that would probably be `(>= 4.0.0-1)`, but since there is no packaged version for that,  it should be correct the way above.